packages/components/pages/f-account-info/package.json

```
@@ -43,10 +43,11 @@
     "@justeat/f-globalisation": "1.0.0"
   },
   "peerDependencies": {
-    "@justeat/browserslist-config-fozzie": ">=1.2.0"
+    "@justeat/browserslist-config-fozzie": ">=1.2.0",
+    "vuex": "3.5.1"
```

This should be set as a range otherwise you're saying that this exact version should be used.

```suggestion
    "vuex": ">=3.5.1"
```